ci/bitrise-complete-guide-cloud-signing.md

```diff
@@ -0,0 +1,183 @@
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/2_4.png"/>
+</p> 
+
+6) The last step is the Webhook setup. This is used to trigger the builds by tags. Tap on **Register a webhook for me**. If the automatic registration does not work, you can [setup the webhook manually](https://devcenter.bitrise.io/en/apps/webhooks/adding-incoming-webhooks.html#adding-incoming-webhooks) now or later. You need to have the admin rights for the Github repository of your app for this though. 
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/2_5.png"/>
+</p> 
+
+## Environment vars setup
+In order to make everything work on Bitrise you'll need to copy the template content from [bitrise.yml](https://github.com/nodes-projects/ci-resources-ios/blob/master/bitrise_cloud_signing.yml) into the Bitrise workflow script.
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/3_1.png"/>
+</p> 
+
+Go to **Workflow > Env** Vars tab. 
+You should see all your environment variables here.
+- SLACK_CHANNEL - name of the Slack channel to send a message when the build succeeds
+- RELEASE_BUILD_CONFIGURATION and TEST_BUILD_CONFIGURATION - change only if you have different names in your project.
+- BITRISE_PROJECT_PATH and BITRISE_SCHEME - change them from placeholders to your project's values. 
+
+Leave the BUILD_CONFIGURATION empty. This is assigned in the preparation steps during building.
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/3_2.png"/>
+</p> 
+
+## Secrets setup
+Go to the **Workflow > Secrets** tab. Add these secrets:
+- BUILD_TRIGGER_API_TOKEN - Can be found in the **Code** tab of your app on Bitrise. This is needed to trigger the build workflows.
+- SLACK_TOKEN - Used to send notifications after successful builds. The token is universal for the Monstarlab Slack Workspace (I think). Can be found [here](https://github.com/nodes-projects/ci-resources-ios/blob/master/slackToken) or copied from another app on Bitrise. 
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/4_1.png"/>
+</p> 
+
+<p align="center">
+  <img src="../images/ci-cloudSigning/4_2.png"/>
+</p> 
+
+## Code signing
+You only need to upload the development signing certificate to Bitrise. This is used to code sign an initial archive before the code signing files are downloaded from Apple and the app is resigned with them. Even this development certificate could be omitted theoretically because the cloud code signing can create a new one if it does not find it locally. This, however, creates multiple new development certificates with every new build so it is better to upload it to Bitrise.
+
+1) Download the certificate from the Apple Developer portal.
+<p align="center">
+  <img src="../images/ci-cloudSigning/5_1.png"/>
+</p>
+
+2) The certificate is in a .cer format. You will need the private key .p12 file though that is embedded inside the .cer file.
```

Comment:
```suggestion
2) The certificate is in a .cer format. You will need the private key .p12 file though that is associated with the .cer file.
```
The private key isn't embedded in the .cer file. Well, technically, the .cer files can contain private keys in them, but Apple's .cer files don't. They're just associated 😊

Reply:
Yes. Correct. Thanky you 💪